aquarium/aquarium.js

```diff
@@ -854,8 +841,43 @@ function initialize() {
   Log("--Setup Laser----------------------------------------");
   var laser = setupLaser();
 
+  num = [1, 100, 500, 1000, 5000, 10000, 15000, 20000, 25000, 30000];
+  var changeViewElem = document.getElementById("setSettingChangeView");
+  var parentElem = changeViewElem.parentNode;
+  for (var i = 0; i < num.length; ++i) {
+    var div = document.createElement("div");
+    div.className = "clickable";
+    div.id = "setSetting" + i;
+    div.innerHTML = num[i];
+    parentElem.insertBefore(div, changeViewElem);
+  }
+
   for (var ff = 0; ff < g_fishTable.length; ++ff) {
     g_fishTable[ff].fishData = [];
+    g_fishTable[ff].num = [];
+  }
+
+  type = ["Big", "Medium", "Small"];
+  for (var i = 0; i < num.length; ++i) {
+    numLeft = num[i];
+    for (var j = 0; j < type.length; ++j) {
+      for (var ff = 0; ff < g_fishTable.length; ++ff) {
+        var fishInfo = g_fishTable[ff];
+        var fishName = fishInfo.name;
+        if (!fishName.startsWith(type[j])) {
+          continue;
+        }
+        if (type[j] == "Big") {
+          numType = Math.min(numLeft, num[i] < 100 ? 1 : 2);
+        } else if (type[j] == "Medium") {
+          numType = Math.min(numLeft, parseInt(num[i] / 10));
```

Comment:
is `parseInt` needed?  Also at 5000 fish there will be 500 medium fish where as before there would have only been 160. I'm not sure it matters

Reply:
parseInt is needed here as fish number may not be multiple of 10, and I hope the number can be easily changed to any value in future. The algorithm of medium fishes is tuned a bit to limit the total number. Is this more reasonable to you?
Other comments have been addressed, and I uploaded a new PR. 
Thanks for the review!

Reply:
If you want an int use 

     Math.min(numLeft, num[i] / 10 | 0);

Reply:
My bad, and parseInt should be used to convert string to integer. Changed to your suggestion. PTAL!